tutorials/in-app/cameraParallax.json

```diff
@@ -0,0 +1,750 @@
+        }
+      },
+      {
+        "messageByLocale": {
+          "en": "- How to create a layer",
+          "fr": "- Comment créer une calque",
+          "es": "- Cómo crear una capa",
+          "pt": "- Como criar uma camada"
+        }
+      },
+      {
+        "messageByLocale": {
+          "en": "- How to add an extension",
+          "fr": "- Comment ajouter une extension",
+          "es": "- Cómo agregar una extensión",
+          "pt": "- Como adicionar uma extensão"
+        }
+      },
+      {
+        "messageByLocale": {
+          "en": "- How to control the camera",
+          "fr": "- Comment contrôler la caméra",
+          "es": "- Cómo controlar la cámara",
+          "pt": "- Como controlar a câmera"
+        }
+      },
+      {
+        "messageByLocale": {
+          "en": "- How to make Tiled Sprites move at different speeds",
+          "fr": "- Comment faire bouger des Tiled Sprites à différentes vitesses",
+          "es": "- Cómo hacer que los Tiled Sprites se muevan a diferentes velocidades",
+          "pt": "- Como fazer Tiled Sprites se movem a diferentes velocidades"
+        }
+      },
+      {
+        "messageByLocale": {
+          "en": "You can keep adding stuff to this game or publish it!",
+          "fr": "Vous pouvez continuer à ajouter des choses à ce jeu ou le publier !",
+          "es": "¡Puedes seguir agregando cosas a este juego o publicarlo !",
+          "pt": "Você pode continuar adicionando coisas a este jogo ou publicá-lo!"
+        }
+      }
+    ]
+  },
+  "flow": [
+    {
+      "id": "Start",
+      "nextStepTrigger": {
+        "clickOnTooltipButton": {
+          "messageByLocale": {
+            "en": "Let's go!",
+            "fr": "C'est parti !",
+            "es": "¡Vamos!",
+            "pt": "Vamos!"
+          }
+        }
+      },
+      "tooltip": {
+        "standalone": true,
+        "description": {
+          "messageByLocale": {
+            "en": "This game is a simple platformer.\nYou can move the player with the arrow keys and jump with space.\n\n**Test it out** with the **preview** button.\n\nWhen you're ready, click on this button to continue.",
+            "fr": "Ce jeu est un simple jeu de plateforme.\nVous pouvez déplacer le joueur avec les flèches et sauter avec Espace.\n\n**Testez-le** avec le bouton **Aperçu**.\n\nQuand vous êtes prêt, cliquez sur ce bouton pour continuer.",
+            "es": "Este juego es un simple plataformero.\nPuedes mover al jugador con las flechas y saltar con la barra espaciadora.\n\n**Pruebalo** con el botón **Previsualizar**.\n\nCuando estés listo, haz clic en este botón para continuar.",
+            "pt": "Este jogo é um simples plataforma.\nVocê pode mover o jogador com as setas e pular com espaço.\n\n**Teste-o** com o botão **pré-visualização**.\n\nQuando estiver pronto, clique neste botão para continuar."
+          }
+        },
+        "placement": "right"
```

Comment:
To remove, it's a bit weird imho
<img width="417" alt="image" src="https://user-images.githubusercontent.com/32449369/224110408-bf80ceea-8e49-439a-a166-f48d57824ad2.png">

Reply:
I think it displays on top by default

Reply:
yes better indeed